scripts/dev_env/5_python_env.sh

```diff
@@ -0,0 +1,53 @@
+if [ -f /usr/bin/virtualenvwrapper.sh ]; then
+    source /usr/bin/virtualenvwrapper.sh
+elif [ -f /usr/share/virtualenvwrapper/virtualenvwrapper.sh ]; then
+    source /usr/share/virtualenvwrapper/virtualenvwrapper.sh
+fi
+
+PYTHON_ENV_NAME=""
+
+# Ask the question for Python environment name
+ask_python_env_name() {
+    print_question "How do you wish to name your Python2 working environment ? [navitia]"
+    local answer
+    read answer
+    if [ -z $answer ]; then
+        PYTHON_ENV_NAME='navitia'
+    else
+        PYTHON_ENV_NAME=$answer
+    fi
+}
+
+# Ask and create Python environment
+ask_python_create() {
+    if [ -z `command -v mkvirtualenv` ]; then
+        print_error "\"mkvirtualenv\" is not available, script is aborting"
+        exit
+    fi
+
+    print_question "Do you wish to set up a Python2 working environment for Navitia? [Y/n]"
+    user_answer
+    if [ $? == 1 ]; then
+        print_info "Configuring Navitia databases ..."
+        python_create_env
+    else
+        print_info "Navitia databases is not configured"
+    fi
+}
+
+# Create Python environment
+python_create_env() {
+    if [ -z $PYTHON_ENV_NAME ]; then
+        print_warn "If the virtual environment already exists, it will be destroyed"
+        ask_python_env_name
+    fi
+
+    rmvirtualenv $PYTHON_ENV_NAME
+    mkvirtualenv -p /usr/bin/python2 $PYTHON_ENV_NAME
+    pip install -r $NAVITIA_HOME/source/eitri/requirements.txt
+    pip install -r $NAVITIA_HOME/source/cities/requirements.txt
+    pip install -r $NAVITIA_HOME/source/tyr/requirements_dev.txt
```

Comment:
We might have different versions requirements depending on the module...
That wouldn't be nice, but that could happen.

And more than that, we want to test as much as possible with the exact requirements that are gonna be deployed.
Anyway, we have to have one venv per requirements.txt (grouping all into one requirement is an option)

Reply:
You're right but it's an help to create a development environment. If you want, you can ignore it ; it's not mandatory to use it. You can create a virtualenv for each project to match the production.

But the risk is acceptable (@nonifier and @benoit-bst agree) for a developper environment. If the issue occurs, the CI should show it.

Reply:
I still disagree.

The CI doesn't launch all the tests, and we don't even have tests on all the components (no cities tests for example).

Also, there are external developers (and production platforms) that could be affected, as it is open-source code.
It's not a personal developer's choice to me, as it can affect anyone maintaining a prod platform: the developer may end up merging things, not knowing that it would crash once deployed. So I think we should not promote it.

It's already more or less the case before that PR given the way we launch docker tests (they mix eitri and tyr tests).

Here is my point, though I'm not blocking the merge.